parsec/interfaces/ptg/ptg-compiler/jdf2c.c

```diff
@@ -7214,9 +7213,22 @@ static void jdf_generate_code_release_deps(const jdf_t *jdf, const jdf_function_
 
     if( !(f->flags & JDF_FUNCTION_FLAG_NO_SUCCESSORS) ) {
 
-        coutput("  arg.output_repo = %s_repo;\n", f->fname);
-        coutput("  arg.output_entry = this_task->repo_entry;\n");
-        coutput("  arg.output_usage = 0;\n");
+        coutput("  parsec_release_dep_fct_arg_t arg;\n"
```

Comment:
This initialization was done all the time and now is done only when FLAG_NO_SUCCESSORS, arg was unused in the report?